.eslintrc

```diff
@@ -262,4 +266,4 @@ rules:
   valid-typeof: [2]
   wrap-iife: [2, inside]
   yield-star-spacing: [2, {before: true, after: false}]
-  yoda: [2, never]
+  yoda: [0]
```

Comment:
Disregard edit.js "style" as it's nonexistent, 99% of this file is inherited from Stylish. There are 15 js files that more or less foloow the modern coding style.